pom.xml

```diff
@@ -78,17 +78,17 @@
         <version.log4j-extras>1.0</version.log4j-extras>
         <version.lucene>7.5.0</version.lucene>
         <version.metrics-cdi>1.6.0</version.metrics-cdi>
-        <version.microservice.accumulo-api>1.1</version.microservice.accumulo-api>
-        <version.microservice.accumulo-utils>1.5</version.microservice.accumulo-utils>
-        <version.microservice.audit-api>1.4</version.microservice.audit-api>
-        <version.microservice.authorization-api>1.13</version.microservice.authorization-api>
-        <version.microservice.base-rest-responses>1.2</version.microservice.base-rest-responses>
-        <version.microservice.common-utils>1.0</version.microservice.common-utils>
-        <version.microservice.dictionary-api>1.0</version.microservice.dictionary-api>
-        <version.microservice.metadata-utils>1.12</version.microservice.metadata-utils>
-        <version.microservice.metrics-reporter>1.2</version.microservice.metrics-reporter>
-        <version.microservice.query-metric-api>1.5.7</version.microservice.query-metric-api>
-        <version.microservice.type-utils>1.11</version.microservice.type-utils>
+        <version.microservice.accumulo-api>2.0-SNAPSHOT</version.microservice.accumulo-api>
```

Comment:
Obviously we will need to change these to non-snapshot versions as soon as we have the other services/utils tagged.

Reply:
as you said, this should be obvious.  These versions are not final.  As we talked about yesterday, I need to merge in Keith's accumulo 2.1 changes which will shuffle all of these versions around.  

Also, given that there is a pretty strict ordering to how these various repos need to be built, I will probably handle this when the time comes to start merging the pull requests.  i.e. I will set the final type-utils version when that PR is merged, publish the release, and update each of the remaining PRs to use the published version.  

But for now, these are more or less placeholder versions.  Let's focus on the logic changes for now.